src/goto-instrument/contracts/utils.h

```diff
@@ -13,6 +13,14 @@ Date: September 2021
 
 #include <goto-programs/goto_convert_class.h>
 
+#include <util/byte_operators.h>
+#include <util/expr_cast.h>
+#include <util/message.h>
+
+#include <goto-programs/goto_model.h>
+
+#include <analyses/dirty.h>
+#include <analyses/locals.h>
```

Comment:
Why do we suddenly need all these includes here?

Reply:
this must be the result of an automatic merge during rebase, will fix